src/librustc/middle/ty.rs

```diff
@@ -5589,14 +5589,8 @@ pub fn predicates<'tcx>(
 }
 
 /// Get the attributes of a definition.
-pub fn get_attrs<'tcx>(tcx: &'tcx ctxt, did: DefId)
-                       -> Cow<'tcx, [ast::Attribute]> {
-    if is_local(did) {
-        let item = tcx.map.expect_item(did.node);
-        Cow::Borrowed(&item.attrs)
-    } else {
-        Cow::Owned(csearch::get_item_attrs(&tcx.sess.cstore, did))
-    }
+pub fn get_attrs<'tcx>(tcx: &'tcx ctxt, did: DefId) -> &'tcx [ast::Attribute] {
+    tcx.map.attrs(did.node)
```

Comment:
You've removed the old logic for cross-crate attributes - only the `expect_item` call needs to be replaced.